sisl/io/tbtrans/tbt.py

```diff
@@ -2742,7 +2743,7 @@ def copy_attr(f, t):
                 v[:] = dvg[:]
 
         # Update the source attribute to signal the originating file
-        self.setncattr('source', 'k-average of: ' + tbt._file)
+        self.setncattr('source', 'k-average of: ' + str(tbt._file))
```

Comment:
casting to `str` should be implicit, I don't think it is needed?

Reply:
I can change it to a string with `format`, but `+` only works if both are strings.